src/style.css

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: column nowrap;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto;
```

Comment:
Which font family will be displaying if google fonts not responding?
Please, add alternative common one, like `Arial` or `sens-serif`.